### PR TITLE
disable code duplication check in unit tests

### PR DESCRIPTION
**Pull request by audrey-jensen**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [X] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Disable code duplication check in unit tests since unit tests duplicate code instead of using control structures.

## Code changes

- **.codescene/code-health-rules.json:** Configures codescene health checks.

### CODE REVIEW

**Comment by audrey-jensen** [comment]
If we want it to apply but minimally, we can instead set the weight of the rule to something low, like "0.1". I imagine if we go that route, that there'll be a bit of tuning involved.

File: .codescene/code-health-rules.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "usage": "Documentation: https://codescene.io/docs/guides/technical/code-health.html. Template: https://codescene.io/projects/26215/config/codehealth/export/code-health-rules.json",
```
Comment (MGibson1) [inline .codescene/code-health-rules.json:2]: :sob: why is this schema locked behind an account???

if you can find one that's not, it'd be great to add a"$scema" hint

**Review by MGibson1 — APPROVED** [review]

File: .codescene/code-health-rules.json (hunk 1 above)
Comment (withinfocus) [reply .codescene/code-health-rules.json:2]: Should we just omit this `usage` property? Seems informational.

**Review by withinfocus — COMMENTED** [review]
Should we import https://codescene.io/projects/26215/config/codehealth/export/code-health-rules.json? It's all default values but would that make the definitions more clear here in the repo? Then you'd add your ruleset below.

File: .codescene/code-health-rules.json (hunk 1 above)
Comment (MGibson1) [reply .codescene/code-health-rules.json:2]: Unfortunately it is, but the information includes the template schema, which is definitely useful for enabling edits in the future.

Comment (audrey-jensen) [reply .codescene/code-health-rules.json:2]: I kept the usage property around to ensure that we can reach the docs easily. If y'all want it removed that's pretty straightforward. I haven't been able to find public versions of those artifacts. Codescene's github doesn't seem to include them.

**Comment by audrey-jensen** [comment]
Following up - the usage property is not an ideal solution, but it's the least wrong solution that links this to documentation. We'd have liked to use a public schema, but there isn't one. We'd have liked to use comments instead of an informational property, but that's prohibited by JSON. So, until a better solution comes around, this is what we've got.

**Pull request merged**